src/components/Header/Header.scss

```
@@ -0,0 +1,22 @@
+.header {
+  display: flex;
+  align-items: center;
+  background-color: aqua;
+  height: 80px;
+  &__link {
+    display: flex;
+    height: 50px;
+    margin-left: 50px;
+    font-size: 20px;
+    border: 1px solid green;
+    color: green;
+    border-radius: 8px;
+    text-decoration: none;
+    align-items: center;
+
+    &-activ {
```

don't forget to group selector's and properties
```suggestion
  display: flex;
  align-items: center;
  
  background-color: aqua;
  
  height: 80px;
  
  &__link {
    display: flex;
    align-items: center;
    
    height: 50px;
    margin-left: 50px;
    
    font-size: 20px;
    
    color: green;
    border: 1px solid green;
    border-radius: 8px;
    
    text-decoration: none;

    &-activ {
```